### PR TITLE
Dev v2.4.0

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Ready for peer review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-## Spectral Indices Version 2.3.0 Release Notes
-Release Date: December 22, 2015
+## Spectral Indices Version 2.4.0 Release Notes
+Release Date: March 23, 2016
```
Comment (ronaldddilley) [inline README.md]: March 23 isn't correct.  I have been leaving the day of month as ?? marks.  And have been thinking about dropping them altogether.

File: src/output.c [hunk 2]
```diff
@@ -124,12 +130,20 @@ Output_t *open_output
...
     bmeta = this->metadata.band;
 
-    /* Determine the scene name */
+    /* Determine the scene name, strip off the reference band information */
     strcpy (scene_name, in_meta->band[refl_indx].file_name);
-    mychar = strchr (scene_name, '_');
-    if (mychar != NULL)
-      *mychar = '\0';
-  
+    mychar = scene_name;
```
Comment (ronaldddilley) [inline src/output.c]: Wouldn't it be easier to use the XML filename and strstr search for ".xml"?  Or maybe we should make the scene name part of the global metadata and just retrieve it from the XML?  Then we only have to determine what it is in product-formatter and all the other places can just use it.

File: src/output.c [hunk 3]
```diff
@@ -90,6 +94,7 @@ Output_t *open_output
         {
             /* this is the index we'll use for reflectance band info */
             refl_indx = ib;
+            strcpy (ref_band_name, "_toa_band1");
```
Comment (ronaldddilley) [inline src/output.c]: low risk, but should use snprintf

File: src/output.c [hunk 4]
```diff
@@ -97,6 +102,7 @@ Output_t *open_output
         { /* check for SR band1 if TOA band1 doesn't exist */
             /* this is the index we'll use for reflectance band info */
             refl_indx = ib;
+            strcpy (ref_band_name, "_sr_band1");
```
Comment (ronaldddilley) [inline src/output.c]: snprintf

**Comment by ronaldddilley** [comment]
No further comments

File: README.md (hunk 1 above)
Comment (gschmidt-usgs) [inline README.md]: I vote for Month and Year vs. adding the day of the month.  That will make
this easier to update for peer reviews and not have to be accurate to the
day we actually release.  Given that this sprint is the March 23rd sprint,
that's the date that was used in the README for release date.

--------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT, Inc.)
Contractor to the USGS Earth Resources Observation and Science (EROS) Center
Rapid City, SD  57701
605-593-2722   Phone
605-394-6078   Fax
gschmidt@usgs.gov

Adaptation - "Manage the unavoidable"
Mitigation - "Avoid the unmanageable"

File: src/output.c (hunk 2 above)
Comment (gschmidt-usgs) [inline src/output.c]: I've been toying with adding the scene name to the global metadata in the
XML and think that will make the filename-related coding easier.

--------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT, Inc.)
Contractor to the USGS Earth Resources Observation and Science (EROS) Center
Rapid City, SD  57701
605-593-2722   Phone
605-394-6078   Fax
gschmidt@usgs.gov

Adaptation - "Manage the unavoidable"
Mitigation - "Avoid the unmanageable"

File: README.md (hunk 1 above)
Comment (gschmidt-usgs) [inline README.md]: Fixed.

File: src/output.c (hunk 2 above)
Comment (gschmidt-usgs) [inline src/output.c]: Pulled scene name from the XML scene ID.

File: src/output.c (hunk 3 above)
Comment (gschmidt-usgs) [inline src/output.c]: Changed non-constant strings to use snprintf.

**Pull request closed without merging**